### PR TITLE
NSObject+Rx compatible with RxSwift 4.0.0

### LINKED ISSUE #51: Support for RxSwift 4.0
Currently had to remove this lib to migrate to RxSwift 4.0

### PR DESCRIPTION
**Pull request by jdisho**
Fixes #51.

### CODE REVIEW

File: NSObject+Rx.podspec [hunk 1]
```diff
@@ -18,5 +18,5 @@ Pod::Spec.new do |s|
   s.source       = { :git => "https://github.com/RxSwiftCommunity/NSObject-Rx.git", :tag => s.version }
   s.source_files  = "*.swift"
   s.frameworks  = "Foundation"
-  s.dependency 'RxSwift', '~> 3.0'
+  s.dependency 'RxSwift', '~> 4.0.0'
```
Comment (ashfurrow) [inline NSObject+Rx.podspec]: Can we keep this as `~> 4.0` instead of `4.0.0` to accommodate version 4.1, 4.2 etc?

**Review by ashfurrow — APPROVED** [review]
This looks good, thanks! Let me know what you think about the comment I left. We can merge once Travis is 🍏  (they're having problems today with their macOS infrastructure).

File: NSObject+Rx.podspec (hunk 1 above)
Comment (jdisho) [reply NSObject+Rx.podspec]: Yes 👍

**Review by freak4pc — COMMENTED** [review]
Looks good, only drop to 4.0 to target minor updates as well.

@ashfurrow should we bump a version as part of this PR ? Or separately ?

**Comment by jdisho** [comment]
@freak4pc @ashfurrow I changed it since this morning 

<sub>Sent with <a href="http://githawk.com">GitHawk</a></sub>

**Comment by ashfurrow** [comment]
Sounds good, I'll merge + release today. Thanks!

**Comment by ashfurrow** [comment]
Thanks a lot for contributing! I've invited you to join the RxSwiftCommunity GitHub organization – no pressure to accept! If you'd like more information on what that means, check out our [contributor guidelines](https://github.com/RxSwiftCommunity/contributors) and feel free to reach out with any questions.

**Comment by ashfurrow** [comment]
Travis was taking forever so I merged manually (see #52). Pushing to CocoaPods now.

**Comment by jdisho** [comment]
@ashfurrow Thank you!! ☺️

**Pull request merged**